src/EditorFeatures/Core.Wpf/InlineErrors/InlineErrorAdornmentManagerProvider.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
Added this because I think the AdornmentLayerDefinition is required but is never set anywhere, so otherwise there is a warning.

Reply:
Probably worth just disabling surrounding that property with a note about this